schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
+create database vet_clinic;
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT,
+    name varchar(100),
```

Comment:
- [ ] [Optional] : It should be better if you mark id column as primary key as follow: `PRIMARY KEY (id)`

Reply:
Okay! Thank You.